driver/config/config.go

```diff
@@ -162,6 +164,7 @@ const (
 	ViperKeyDatabaseCleanupBatchSize                         = "database.cleanup.batch_size"
 	ViperKeyLinkLifespan                                     = "selfservice.methods.link.config.lifespan"
 	ViperKeyLinkBaseURL                                      = "selfservice.methods.link.config.base_url"
+	ViperKeyCodeLifespan                                     = "selfservice.methods.code.config.lifespan"
```

Comment:
In our discussions I recall that we agreed on the following config layout:


```
selfservice:
  flows:
    verification:
      use: code
    recovery:
      use: code
```

instead of validating:


```
selfservice:
  methods:
    code:
      enabled: true|false
    link:
      enabled: true|false
```

because it makes sense to have only one type enabled at a single time.

Reply:
Yes, that's true and this layout looks a lot easier to use. 

Initially, I didn't want to break the current mechanisms we have in place for the relation of methods/strategies -> flow (and not the other way around). But your proposal looks way cleaner. Will do.

Reply:
Done.

Although, I did not rewrite the logic to enable/disable strategies, as that is sufficiently generic. Now you can use `selfservice.flows.recovery.use: code` to use the new strategy. It is enabled by default, which means you'd only need to explicitly enable the link strategy if you want to use it.

Reply:
great!